gcc/rust/backend/rust-compile-pattern.h

```diff
@@ -117,13 +117,13 @@ class CompilePatternLet : public HIRCompileBase, public HIR::HIRPatternVisitor
   void visit (HIR::WildcardPattern &) override;
   void visit (HIR::TuplePattern &) override;
 
-  // check for unimplemented Pattern HIR nodes.
   void visit (HIR::AltPattern &pattern) override
   {
-    rust_sorry_at (pattern.get_locus (),
-		   "alternate pattern let statements not supported");
+    rust_error_at (pattern.get_locus (),
+		   "top-level or-patterns are not allowed in %<let%> bindings");
```

Comment:
It's fine to use an error message different from `rustc` that suits our compiler logic better :) this is not a critic or anything, just something to keep in mind when you think about new errors

Reply:
Ok